exercises/practice/clock/src/clock.cr

```diff
@@ -0,0 +1,71 @@
+class Clock
+  HOURS_PER_DAY    = 24
```

Comment:
since this file is provided to the students, it would leave the students with no work to do if this file already contained a complete solution. all functionality should be removed from this file, leaving only the signatures of the public functions that will be tested by the test suite. Those should be just the function declaration and parameters, with no body